### PR TITLE
Cal route

### PR DESCRIPTION
**Pull request by mpolimen**

### CODE REVIEW

**Comment by mpolimen** [comment]
The last two commits were an attempt to get around github not tracking empty folders, not trying to inflate commits.

File: src/routes/board.js [hunk 1]
```diff
@@ -83,6 +85,49 @@ const equalMembers = (l1, l2) => {
...
+        }
+        if(!foundUser)
+            throw new Error("Not your board, therefore can not see card.")
+    } catch(e) {
+        res.status(403).render('error-page', { title: "403 Forbidden", message: e.toString(), error: true });
+        return;
+    }
+
+    try {
+        console.log(boardInfo._id);
```
Comment (roccopolimen) [inline src/routes/board.js:122]: remove, unnecesary

**Review by roccopolimen — COMMENTED** [review]
looks good, will test properly with the front end application when merged.

**Comment by roccopolimen** [comment]
I also think downloading calendar should be a router.get instead of a router.post so another simple tweak.

**Comment by bwormser** [comment]
Looks pretty solid

**Pull request merged**